### PR TITLE
Audit

### PR DESCRIPTION
**Pull request by kaylaewood**
- [x] Organized Repo
- [x] Audited all test suites
- [x] Deleted irrelevant or incomplete files
- [x] Moved other test suites into this repo (golf, skate park, lunchbox, oregon trail, crafting)
- [x] Rewrote READMEs to be consistent and accurate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,5 +28,28 @@ You should see something like:
   0 passing (6ms)
   7 pending
 ```
+5. Make sure you read each subdirectory's README carefully before starting to code!
+
+## Overview
+
+There are lots of test suites in this repo! Our recommendation is to complete each **Mythical Creature** before the End of Module Assessment. After you've completed all of the creatures, start working through the other test suites listed below (in any order). We find that when students complete the same test suite over and over, memorization and pattern matching kick in (which is not what we want!). These test suites should be improving your problem solving skills. If you feel yourself being able to write code without reading the full test, it’s probably time for a new test suite. You can (and should) be strategic about how you use these - for example, you might use one of the repos for a mock final assessment with your mentor.
+
+First Priority:
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply README.md:37]: I thought about this too -- I can go either way! What do you think?

File: golf-course/golf-course.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class GolfCourse {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply golf-course/golf-course.js:1]: good catch!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hannahhch) [reply README.md:37]: It probably wouldn't hurt! That way we're less likely to lose track of them.

File: iterator-factory/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+In order to be able to run these tests you need to install the dependencies at the root level of this repository. Find the directions [here]([root-level of this repository](https://github.com/turingschool-examples/javascript-foundations)).
+
+### Running the Tests
+
+You can run the tests from the root of this repository. When you type `pwd` in the console, the last path location should be `javascript-foundations`.
+
+To run the test file, enter the following command in your terminal.
+
+```shell
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply iterator-factory/README.md]: done!

File: iterator-factory/lib/iteratorFactory.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+class IteratorFactory {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply iterator-factory/lib/iteratorFactory.js]: done!

File: lunchbox/snack-test.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  it('should see stock decrease after eaten', function() {
+    var pistachio = new Snack("Pistachio");
+
+    pistachio.getEaten();
+    pistachio.getEaten();
+    pistachio.getEaten();
+    assert.equal(pistachio.amount, 70);
+  });
+
+  it('should be cutting it close if 20% or left remaining', function() {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply lunchbox/snack-test.js:41]: done

File: oregon-trail/test/gear-test.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  })
+
+  it('should be able to have a different type', function() {
+    var ammunition = new Gear('ammunition');
+
+    assert.equal(ammunition.type, 'ammunition');
+  })
+
+  // valid types are food, ammunition, and clothes
+  it('should not be able to have an invalid type', function() {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply oregon-trail/test/gear-test.js:18]: yes - this is why OT is so difficult. I agree that we should rewrite this

File: wizarding-arrays/test/potion-test.js [hunk 7]
```diff
@@ -11,19 +11,19 @@ describe('A wizard\'s potions cabinet and potions', () => {
       assert.equal(potion.name, 'truth serum');
     });
 
-    it.skip('indicates rareness by integers, 0 being not rare', () => {
+    it('indicates rareness by integers, 0 being not rare', () => {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-arrays/test/potion-test.js:14]: I think fat arrows are okay for them to see?

File: wizarding-arrays/test/spell-test.js [hunk 8]
```diff
@@ -28,7 +28,7 @@ describe('Wizard spells', () => {
     assert.equal(spell.execute(targets), "Success! The book is levitating. Success! The table is levitating. Success! The bowl is levitating.");
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-arrays/test/spell-test.js:28]: yeah agreed - it seems like it's not dynamic. We can change this

File: lunchbox/snack-test.js [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    pistachio.getEaten();
+    pistachio.getEaten();
+    pistachio.getEaten();
+    pistachio.getEaten();
+    assert.equal(pistachio.cuttingItClose, false);
+    pistachio.getEaten();
+    assert.equal(pistachio.cuttingItClose, true);
+  });
+
+  it('should be considered healthy if the word "fruit" is in it', function() {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply lunchbox/snack-test.js:56]: done!

File: wizarding-arrays/test/potion-test.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (hannahhch) [reply wizarding-arrays/test/potion-test.js:14]: Okay, yeah I'm good with that!

File: wizarding-objects/test/wand-test.js [hunk 10]
```diff
@@ -3,39 +3,39 @@ const Wand = require('../lib/wand');
 
 describe('Wand', () => {
 
-  it.skip('should be a function', () => {
+  it('should be a function', () => {
     const wand = new Wand('Elm');
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-objects/test/wand-test.js:7]: done!

File: lunchbox/snack-test.js [hunk 11]
```diff
@@ -0,0 +1,69 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply lunchbox/snack-test.js:1]: done

File: lunchbox/lunchbox-test.js [hunk 12]
```diff
@@ -0,0 +1,86 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply lunchbox/lunchbox-test.js:1]: done

File: oregon-trail/test/gear-test.js [hunk 13]
```diff
@@ -0,0 +1,38 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply oregon-trail/test/gear-test.js:1]: done!

File: oregon-trail/test/part-test.js [hunk 14]
```diff
@@ -0,0 +1,65 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply oregon-trail/test/part-test.js:1]: done!

File: oregon-trail/test/settler-test.js [hunk 15]
```diff
@@ -0,0 +1,117 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply oregon-trail/test/settler-test.js:1]: done~

File: oregon-trail/test/wagon-test.js [hunk 16]
```diff
@@ -0,0 +1,302 @@
+var assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply oregon-trail/test/wagon-test.js:1]: done

File: wizarding-arrays/test/spell-test.js [hunk 17]
```diff
@@ -9,14 +9,14 @@ describe('Wizard spells', () => {
     assert.equal(spell.description, 'makes things levitate');
   });
 
-  it.skip('can be executed', () => {
+  it('can be executed', () => {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-arrays/test/spell-test.js]: done

File: wizarding-objects/test/pet-test.js [hunk 18]
```diff
@@ -3,23 +3,23 @@ const Pet = require('../lib/pet');
 
 describe('Pet', () => {
 
-  it.skip('should be a function', () => {
+  it('should be a function', () => {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-objects/test/pet-test.js]: done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaylaewood) [reply README.md:37]: great! I'll do it!

File: trick-or-treat/README.md [hunk 19]
```diff
@@ -0,0 +1,20 @@
+# Objects and Functions
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply trick-or-treat/README.md:1]: done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaylaewood) [reply README.md:37]: done~

File: wizarding-arrays/test/spell-test.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (kaylaewood) [reply wizarding-arrays/test/spell-test.js]: I changed to say `Success! You've cast a spell on the book. Success! You've cast a spell on the table. Success! You've cast a spell on the bowl.`

File: golf-course/golf-course-test.js [hunk 20]
```diff
@@ -0,0 +1,166 @@
...
+    assert.equal(golfer.handicap, 21);
+  });
+
+  it.skip('should start out minimally frustrated', function () {
+    var golfer = new Golfer({ name: 'Ryan', handicap: 19 });
+    assert.equal(golfer.name, 'Ryan');
+    assert.equal(golfer.frustration, 0);
+  });
+
+  it.skip('should be able to calculate their average score based on par 72', function () {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply golf-course/golf-course-test.js:86]: done!

File: golf-course/golf-course-test.js [hunk 21]
```diff
@@ -0,0 +1,166 @@
...
+  it.skip('should have some unique features', function () {
+    var golfCourse1 = new GolfCourse('Arrowhead', 'hard', 42, ['great views', 'wildlife']);
+    var golfCourse2 = new GolfCourse('Bay Hill', 'hard', 11, ['water', 'Arnies Place']);
+    assert.deepEqual(golfCourse1.features, ['great views', 'wildlife']);
+    assert.deepEqual(golfCourse2.features, ['water', 'Arnies Place']);
+  });
+
+  // PASS GOLFER TESTS BELOW BEFORE FINISHING REMAINING GOLF COURSE TESTS
+
+  it.skip('should accept golfers if there is room available', function () {
```
[the remark this answers is not in the document]
Comment (kaylaewood) [reply golf-course/golf-course-test.js:28]: This test does test that -- the group isn't able to be checked in

[the remark this answers is not in the document]
Comment (kaylaewood) [reply golf-course/golf-course-test.js:28]: I'll separate into two tests!

**Pull request merged**